### PR TITLE
Fixes to integrate oidc provider

### PR DESCRIPTION
**Pull request by Jasopaum**

### CODE REVIEW

File: example-client/hydra/compose-hydra.yml [hunk 1]
```diff
@@ -13,15 +13,15 @@ services:
       - URLS_SELF_ISSUER=http://localhost:4444
       - URLS_CONSENT=http://localhost:3002/consent
       - URLS_LOGIN=http://localhost:3002/login
-      - URLS_LOGOUT=http://localhost:3002/logout
+      # - URLS_LOGOUT=http://localhost:3002/logout
```
Comment (simonvadee) [inline example-client/hydra/compose-hydra.yml:16]: remove

File: example-client/hydra/compose-hydra.yml [hunk 2]
```diff
@@ -13,15 +13,15 @@ services:
...
       - DSN=memory
       - SECRETS_SYSTEM=youReallyNeedToChangeThis
       - OIDC_SUBJECT_IDENTIFIERS_SUPPORTED_TYPES=public,pairwise
       - OIDC_SUBJECT_IDENTIFIERS_PAIRWISE_SALT=youReallyNeedToChangeThis
       - LOG_LEAK_SENSITIVE_VALUES=true
       - CORS_ENABLED=true
       - CORS_ALLOWED_ORIGINS=*
-      - ACCESS_TOKEN_LIFESPAN=10s
+      # - ACCESS_TOKEN_LIFESPAN=10s
```
Comment (simonvadee) [inline example-client/hydra/compose-hydra.yml:24]: remove

File: main.go [hunk 3]
```diff
@@ -21,10 +21,12 @@ const sessionName = "authentication"
 func main() {
 
 	loginRequestRoute := env.Getenv("LOGIN_REQUEST_ROUTE", "http://localhost:4445/oauth2/auth/requests/login")
+	// logoutRequestRoute := env.Getenv("LOGOUT_REQUEST_ROUTE", "http://localhost:4445/oauth2/auth/requests/logout")
```
Comment (simonvadee) [inline main.go:24]: why commented ?

File: main.go [hunk 4]
```diff
@@ -21,10 +21,12 @@ const sessionName = "authentication"
...
 	consentRequestRoute := env.Getenv("CONSENT_REQUEST_ROUTE", "http://localhost:4445/oauth2/auth/requests/consent")
 
 	hConf := &provider.HydraConfig{
-		LoginRequestRoute:   loginRequestRoute,
+		LoginRequestRoute: loginRequestRoute,
+		// LogoutRequestRoute:  logoutRequestRoute,
```
Comment (simonvadee) [inline main.go:29]: same

File: main.go [hunk 5]
```diff
@@ -53,5 +57,6 @@ func main() {
 	// Start http server
 	serverUrl := fmt.Sprintf("0.0.0.0:%s", env.Getenv("PORT", "3002"))
 	fmt.Printf("Listening on: %s\n", serverUrl)
+	fmt.Printf("root url: %s\n", env.Getenv("ROOT_URL", ""))
```
Comment (simonvadee) [inline main.go:60]: remove?

File: main.go [hunk 6]
```diff
@@ -44,6 +46,8 @@ func main() {
 	// Set up a router and some routes
 	router.GET("/login", envContext.GetLogin)
 	router.POST("/login", envContext.PostLogin)
+	// router.GET("/logout", envContext.GetLogin)
```
Comment (simonvadee) [inline main.go:49]: ?

**Review by simonvadee — CHANGES_REQUESTED** [review]

File: main.go (hunk 3 above)
Comment (Jasopaum) [reply main.go:24]: The logout flow is still not implemented, I'm not sure what it's supposed to do and not 100% clear on how it works
Currently, we revoke tokens at logout which should be sufficient

**Review by simonvadee — APPROVED** [review]

**Pull request merged**